src/constructs/loadbalancing/alb.test.ts

```diff
@@ -1,4 +1,5 @@
 import "@aws-cdk/assert/jest";
```

Comment:
A lot of the tests in this file do the same thing, so have deleted the duplicates.